driver/pelican/demandResponse.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"time"
+)
+
+type DR_EVENT_STATUS int
+
+const (
+	DR_EVENT_UNKNOWN        DR_EVENT_STATUS = -1
```

Comment:
Strictly speaking, -1 is not a valid value for either the DR event status or type in the current specification of the XBOS interface.

Do we want to remove those values here or update the interface spec to incorporate them?

Reply:
Sounds good! I'll make sure to return an fmt.Error as opposed to assigning an actual error. In that case, I will probably just removed the "unknown" and "unrecognized" cases for the event status and type structs.